gcp/rakefiles/entrypoint.rake

```diff
@@ -82,6 +83,30 @@ task :deploy => [:set_vars, @gcp_creds_file, @serviceaccount_key_file, @kubectl_
   sh "#{@exekube_cmd} up"
 end
 
+desc "[ADVANCED] Deploy provided ENV['RES'] module into the cluster"
```

Comment:
Sergey clarified that these are convenience methods. They're not required, which is why normal terragrunt dependencies are not the answer.

Sergey, can you please move these toward the end of the file near other convenience methods that are not part of the standard workflow, like :xk.